README.md

```diff
@@ -98,3 +107,14 @@ tooling and integrations with StrongLoop features:
     Switch between npm registries and promote packages between them. Useful
     as a component of DevOps/CI pipelines, or from the command line.
 ```
+
+
+## License
+
+strongloop uses a dual license model.
+
+You may use this library under the terms of the [Artistic 2.0 license][],
```

Comment:
Might want to add a line about the dependencies each being under their own licenses, just to be clear.